pip/compat.py

```diff
@@ -41,7 +46,14 @@ def console_to_str(s):
         try:
             return s.decode(sys.__stdout__.encoding)
         except UnicodeDecodeError:
-            return s.decode('utf_8')
+            if WINDOWS:
+                try:
+                    from ctypes import cdll
+                    return s.decode("cp" + str(cdll.kernel32.GetACP()))
```

Comment:
According to docs the backslashreplace error handling is only for encoding,

So I think maybe we could just change the code from
`return s.decode("cp" + str(cdll.kernel32.GetACP()))`
to
`return s.decode("cp" + str(cdll.kernel32.GetACP()), "replace")`
to prevent error.